client/src/test/scala/dwolla/cloudflare/ZoneSettingsClientSpec.scala

```diff
@@ -119,7 +119,11 @@ class ZoneSettingsClientSpec(implicit ee: ExecutionEnv) extends Specification wi
           |      "name_servers": [
           |        "eric.ns.cloudflare.com",
           |        "lucy.ns.cloudflare.com"
-          |      ]
+          |      ],
```

Comment:
Could this be converted to a JSON literal instead of a string, so the JSON is checked at compile time?

Reply:
Addressed.